grc/dab_ofdm_sync_dab2.xml

```diff
@@ -4,12 +4,17 @@
   <key>dab_ofdm_sync_dab2</key>
   <category>[DAB]</category>
   <import>import dab</import>
-  <make>dab.ofdm_sync_dab2($dab_params, $rx_params, $debug)</make>
+  <make>dab.ofdm_sync_dab2(dab.parameters.dab_parameters(mode=$mode, sample_rate=$samp_rate, verbose=False), dab.parameters.receiver_parameters(mode=$mode), False)</make>
```

Comment:
This is ever so slightly lacking of beauty.

If possible, can you add a `from parameters import dab_parameters` into the __init__.py, thus enabling you to use `dab.dab_parameters` instead?